babel-code-frame/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this is reseved for depending on other packages that ship thier declaration files, e.g. moment, ng2, etc..